### PR TITLE
Add Dennis Mathenge

### PR DESCRIPTION
**Pull request by creativehubspace**

### CODE REVIEW

**Review by gatarelib — COMMENTED** [review]
The page is yet to be populated with data I guess.

**Comment by creativehubspace** [comment]
Working on it.

**Comment by creativehubspace** [comment]
Added a few details to the page.

File: src/data.js [hunk 1]
```diff
@@ -40,6 +40,28 @@ module.exports = [
       'TailwindCSS',
     ],
   },
+  {
+    name: 'Dennis Mathenge',
+    description: 'Web Developer',
+    url: 'https://creativehubspace.com/uses',
+    twitter: '_dennisnm',
```
Comment (davidak) [inline src/data.js:47]:
```suggestion
    twitter: '@_dennisnm',
```
this is probably needed to work

**Review by davidak — CHANGES_REQUESTED** [review]

File: src/data.js (hunk 1 above)
Comment (creativehubspace) [reply src/data.js]: Twitter handle updated.

**Review by davidak — APPROVED** [review]
LGTM

**Pull request merged**